README.md

```diff
@@ -1,2 +1,175 @@
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> This is a website I created using basic HTML and CSS. The aim is to configure linters on the local enviroment and ensure there are no Linter errors.
+
+## 🛠 Built With <a name="built-with"></a>
+
+1- HTML
+2- CSS
+3- Linters
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.learn-html.org/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Added index.html file.
+- Added style.css file.
+- Added linter files and folders.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Not yet available at present, coming soon.
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps below.
+
+### Prerequisites
+
+In order to run this project you need:
+1- Web Browser
+2- Code Editor
+3- Git
+
+Example command:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git https://github.com/Mohau-Moloi/Hello-Microverse.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Open the index.html file utilizing the live server extension in your code editor.
+
+### Run tests
+
+> Feature not yet available, coming soon.
+
+### Deployment
+
+> Feature not yet available, coming soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Mohau Moloi
+
+- GitHub: [@githubhandle](https://github.com/Mohau-Moloi)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add Header
+- [ ] Add Navigational Bar
+- [ ] Add Footer
+- [ ] Add more CSS
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project leave a comment.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly fix the MIT License link in your READE file. It is broken at this moment. You have MIT.md file in your project not LICENSE file. Kindly look into that and fix.

### License
![mit](https://github.com/Mohau-Moloi/Hello-Microverse/assets/26597432/bc4bea34-a76b-4977-a37d-8f2c91c7ef7a)